src/server/rdb_load.cc

```diff
@@ -733,13 +733,14 @@ void RdbLoaderBase::OpaqueObjLoader::CreateList(const LoadTrace* ltrace) {
     if (ec_)
       return false;
 
+    uint8_t* lp = nullptr;
     if (container == QUICKLIST_NODE_CONTAINER_PLAIN) {
-      quicklistAppendPlainNode(ql, (uint8_t*)sv.data(), sv.size());
+      lp = (uint8_t*)zmalloc(sv.size());
```

Comment:
ouch!

Reply:
?

Reply:
I think that Roman is implying that this is a nasty bug :)